src/components/SideBar/SideBarComponents/searchComponet.js

```diff
@@ -86,6 +85,9 @@ function localSearchCOmponent({ classes }) {
     localSearch([]);
     return null;
   }
+  const filterOptions = createFilterOptions({
```

Comment:
Are we planning to add some more functions here? otherwise, you can directly pass this func in parameter directly.